tests/installation/module_selection/select_modules_desktop_sdk.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Register nonconflicting modules:
+# - Containers Module
+# - Desktop Application Module
+# - Development Tools Module
+# - Legacy Module
+# - Transactional Server Module
+# - Web and Scripting Module
```

Comment:
I think here we should only have the `desktop` and `development` modules

Reply:
Thanks George, failed for the copy&paste again. ;-)